container/libcontainer/handler.go

```diff
@@ -758,6 +760,15 @@ func (h *Handler) GetProcesses() ([]int, error) {
 	return pids, nil
 }
 
+func (h *Handler) ReadSmaps() error {
+	pids, err := h.cgroupManager.GetPids()
+	for {
+		h.referencedMemory, err = referencedBytesStat(pids, h.cycles, *referencedResetInterval)
+		time.Sleep(10 * time.Second)
```

Comment:
That should be configurable or maybe we can make it dynamic/adaptive depending on time needed to read that data.

We could achieve trade-off between accuracy and overhead/intrusiveness:
- for small processes, there will be small overhead but better precision,
- for larger processes (>10GB), the results will be less accurate but with smaller CPU overhead

Just an idea.